packages/app/src/components/Header/index.tsx

```diff
@@ -137,28 +192,37 @@ const Header = ({ data }: { data: any }) => {
                 }}>
                   Show History
               </MenuItem>
-              <MenuItem onClick={() => {
-                  if (isProvVisOpen) {
-                    setIsProvVisOpen(false);
-                  }
-                  setIsElementSidebarOpen(true); 
-                  handleMenuClose();
-                }}>
-                  Show Element View
-              </MenuItem>
             </Menu>
-          <Button
+          <IconButton
             color="inherit"
-            sx={{ minWidth: "24px" }}
-            onClick={() => {
-              oAuth.logout();
-              if (window) {
-                window.location.href = getMultinetDataUrl(workspace);
-              }
+            sx={{ width: "32px", height: "32px" }}
+            onClick={(e) => {
+              handleLoginOpen(e);
             }}
           >
-            <AccountCircle color="inherit"></AccountCircle>
-          </Button>
+            <Avatar sx={{ width: "32px", height: "32px" }} alt="User login status icon" variant="circular">
+              {userInfo !== null ?
+                `${userInfo.first_name.charAt(0)}${userInfo.last_name.charAt(0)}`
+                : <AccountCircle sx={{ height: "90%", width: "90%"}} color="inherit" />
+              }
+            </Avatar>
+          </IconButton>
```

Comment:
I seem to remember seeing some issue with this, circles not being fully concentric. I might be tripping though. Could you attach a photo of the icon when logged in and out? Perhaps we can make it look a bit more like the vuetify version for a more cohesive feel

Reply:
Logged in:
![image](https://github.com/visdesignlab/upset2/assets/35744963/513ebf87-e1c9-4007-a7e0-2a90523374be)

Logged out:
![image](https://github.com/visdesignlab/upset2/assets/35744963/38ac6ed4-43f3-4278-a0a3-06448c80c162)